app/models/review.rb

```diff
@@ -0,0 +1,6 @@
+class Review < ApplicationRecord
+  belongs_to :podcast
+
+  validates :rating, presence: true, inclusion: { in: 1..5 }
+  validates :body, presence: true, length: { maximum: 500, minimum: 5 }
```

Comment:
500 characters seems a bit short - are you sure you want to limit to that length?